tests/conftest.py

```diff
@@ -22,49 +36,62 @@
 )
 
 MASTER_KEY = "masterKey"
-BASE_URL = "http://127.0.0.1:7700"
+BASE_URL = "https://127.0.0.1:7700"
```

Comment:
Change this to a fixture set based on command line arguments and change references to `BASE_URL` to use the fixture. See larger discussion round ssl as to why.

```suggestion
def pytest_addoption(parser):
    parser.addoption("--http2", action="store_true")


@pytest.fixture(scope="session")
def base_url(request):
    if request.config.getoption("--http2"):
        return "https://127.0,0.1"

    return "http://127.0.0.1"
```

Reply:
i split it into http2_enabled fixture and base_url fixture

Reply:
Yep, makes sense